promptsource/templates/math_qa/templates.yaml

```diff
@@ -62,25 +115,40 @@ templates:
 
       '
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_2
-    reference: ''
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      original_task: false
+    name: problem_set_type
+    reference: The template asks to generate the category of the problem set
   b4881325-de90-4cf5-8100-d4a2e13aa017: !Template
     answer_choices: null
     id: b4881325-de90-4cf5-8100-d4a2e13aa017
-    jinja: 'Given the problem below, what is the {{"linear formula"}} you would use
-      to solve it?
+    jinja: 'Given the problem below, what is the linear formula you would use to solve
+      it?
 
       ===
 
       {{Problem}} |||
 
       {{linear_formula}}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
-      original_task: null
-    name: Template_4
-    reference: ''
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: linear_formula
```

Comment:
this prompt is extremely weird. it doesn't give the model the possible operations (multiply, add, const, etc...), so I'd expect to have a 0 perf... same for `annotated_formula`.
either we remove it, or you can write instructions on which operations to use...